public/sw.js

```
@@ -0,0 +1,353 @@
+      ("document" in self ? document.currentScript.src : "") ||
+      location.href;
+    if (e[c]) return;
+    let a = {};
+    const r = (s) => t(s, c),
+      f = { module: { uri: c }, exports: a, require: r };
+    e[c] = Promise.all(i.map((s) => f[s] || r(s))).then((s) => (n(...s), a));
+  };
+}
+define(["./workbox-3cafb6cd"], function (s) {
+  "use strict";
+  (importScripts(),
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)